mcp-server/src/core/direct-functions/add-task.js

```diff
@@ -8,15 +8,6 @@ import {
 	enableSilentMode,
 	disableSilentMode
 } from '../../../../scripts/modules/utils.js';
-import {
-	getAnthropicClientForMCP,
-	getModelConfig
-} from '../utils/ai-client-utils.js';
-import {
-	_buildAddTaskPrompt,
-	parseTaskJsonResponse,
-	_handleAnthropicStream
-} from '../../../../scripts/modules/ai-services.js';
```

Comment:
nice, we should probably start adding an extra layer by adding a linter, this way we remove unused variables, and start enforcing a more strict code style to make our codebase easier.